scylla-artifacts.py

```diff
@@ -261,6 +261,11 @@ def run(self):
                          '(see logs for details)' % os.path.basename(pkg))
                 raise InstallPackageError(e_msg)
         process.run('/usr/lib/scylla/scylla_io_setup', shell=True)
+
+        uuid_path = '/etc/scylla.d/housekeeping.uuid'
+        url = 'https://i6a5h9l1kl.execute-api.us-east-1.amazonaws.com/prod/check_version?uu=$id&mark=scylla'
+        if os.path.exists(uuid_path):
+            process.run('id=`cat %s`;curl "%s"' % (uuid_path, url), shell=True)
```

Comment:
@lmr I did that in scylladb/scylla-cluster-tests/pull/217, I will update this PR to be more pythonic.